### PR TITLE
chore: showing powere by thirdweb on thirdweb minttypes

### PR DESCRIPTION
**Pull request by ximxim**
## Description

Showing powered by thirdweb on success state of mintdialog for mintTypes of thirdweb

## Deploy Notes

Notes regarding deployment of the contained body of work. These should note any
db migrations, nginx routes, infrastructure changes, and anything that must be done before deployment.

- [ ] Need to add environment variables
  - `ENV_VAR=`

## Screenshots (if appropriate)

| Before | After |
| ------ | ----- |
| image | image |

### CODE REVIEW

File: src/components/icons/ThirdWebPill.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <rect
+        y="0.703613"
+        width="81"
+        height="22.5929"
+        rx="11.2965"
+        fill="#EFEFEF"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```
Comment (lauchness) [inline src/components/icons/ThirdWebPill.tsx:27]:
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```

Comment (lauchness) [reply src/components/icons/ThirdWebPill.tsx:27]: Avoid console errors

File: src/components/MintDialog/pages/Success/Success.tsx [hunk 2]
```diff
@@ -59,6 +61,11 @@ export const Success: FC<SuccessProps> = ({
       <div>
         <h3 className="font-sans text-[16px] text-[#151515]">Share on </h3>
         <Share />
+        {mintType === MintType.ThirdWeb && (
+          <p className="text-sm font-mono text-[#5B616E] mt-6 flex items-center">
+            Mint contract powered by <ThirdWebPill className="inline ml-2" />
```
Comment (lauchness) [inline src/components/MintDialog/pages/Success/Success.tsx:66]: could chuck a sr-only span in here saying "ThirdWeb"

```suggestion
            Mint contract powered by <span className="sr-only">ThirdWeb</span><ThirdWebPill className="inline ml-2" />
```

**Comment by cb-heimdall** [comment]
Review Error for lauchness @ 2023-08-17 14:10:42 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Pull request merged**